examples/CMakeLists.txt

```diff
@@ -177,6 +182,31 @@ add_subdirectory(nodeset)
 #  Example PubSub  #
 ####################
 
+if(UA_ENABLE_PUBSUB)
+    if(UA_ENABLE_PUBSUB_CUSTOM_PUBLISH_HANDLING)
+        include_directories(${CMAKE_SOURCE_DIR}/pubsub_realtime/udp_socket)
+        include_directories(${CMAKE_SOURCE_DIR}/include)
+        include_directories(${CMAKE_SOURCE_DIR}/plugins)
+        include_directories(${CMAKE_SOURCE_DIR}/plugins/include)
+        include_directories(${CMAKE_BINARY_DIR})
+        include_directories(${CMAKE_BINARY_DIR}/src_generated)
+        include_directories(${CMAKE_SOURCE_DIR}/arch)
+        include_directories(${CMAKE_SOURCE_DIR}/arch/posix)
+        include_directories(${CMAKE_SOURCE_DIR}/examples)
+        include_directories(${CMAKE_SOURCE_DIR}/src/server)
+        include_directories(${CMAKE_SOURCE_DIR}/src)
+        include_directories(${CMAKE_SOURCE_DIR}/src/pubsub)
+        include_directories(${CMAKE_SOURCE_DIR}/deps)
```

Comment:
To avoid problems when `open62541` is included in other  projects via `add_subdirectory`, it is recommended to use `PROJECT_SOURCE_DIR` instead of `CMAKE_SOURCE_DIR`, see https://github.com/open62541/open62541/issues/2436 .